jwst/associations/lib/dms_base.py

```diff
@@ -753,6 +798,28 @@ def __init__(self, *args, **kwargs):
         )
 
 
+class Constraint_Coron(Constraint):
+    """Match on Coronagraphic Observations"""
+    def __init__(self, *args, **kwargs):
+        super(Constraint_Coron, self).__init__(
+            [
+                DMSAttrConstraint(
+                    sources=['pntgtype'],
+                    value='science'
+                ),
+                Constraint(
```

Comment:
Should not need the wrapping `Constraint`. The DMSAttrConstraint alone should be fine.

The logic is that any value of CORON_EXP_TYPES is good, and whichever is chosen, will be the one used for this association.

Reply:
fixed in latest commit